Fix for simple server turnout to return status when thrown/closed not…

AlgerP572 opened the pull request:
… specified.

Hi.  I have been setting up raspberry pi's for remote access to turnouts and sensors.  While setting things up I noticed that an exception was occurring when trying to retrieve status of the turnout.  The proposed change below seems to fix the issue.  I am not a java expert so if there is a different approach to fix this issue please let me know.  The issue is that when the client asks for status there is not another space in the statusString and the index of returns a -1...

geowar1 reviewed (commented):
That's a lot of code for something you could do with one line:
String turnoutName = statusString.split(" ")[1];

AlgerP572 commented:
> That's a lot of code for something you could do with one line:
> String turnoutName = statusString.split(" ")[1];

Good call!  After making this change I also changed a couple of lines below to use turnoutName.  This way the original index variable could be removed as well.  I've tested it locally here on my system seems to work...
